quixstreams/models/serializers/avro.py

```diff
@@ -0,0 +1,58 @@
+from typing import Union, Mapping, Optional, Any, Iterable
+
+from io import BytesIO
+
+from fastavro import schemaless_reader, schemaless_writer, parse_schema
+from fastavro.types import Schema
+
+from .base import Serializer, Deserializer, SerializationContext
+from .exceptions import SerializationError
+
+__all__ = ("AvroSerializer", "AvroDeserializer")
+
+
+class AvroSerializer(Serializer):
+    def __init__(self, schema: Schema, **kwargs):
```

Comment:
About arguments pass-through: 
I think it's better to expand all (or most of) the arguments in the list and pass them directly.

It will give users more information about the behavior without necessarily visiting the other library's docs, and it will give us more control in case we need to do something with these arguments in the future.